kebechet/managers/version/utils.py

```diff
@@ -205,11 +204,29 @@ def _is_merge_event(payload: Dict[str, Any]):
             "This function only works for GitHub payloads at this time."
         )
     if (
-        payload["event"] == "pull_request"
-        and payload["raw_payload"]["payload"]["action"] == "closed"
+            payload["event"] == "pull_request"
+            and payload["raw_payload"]["payload"]["action"] == "closed"
     ):
         return payload["raw_payload"]["payload"]["pull_request"]["merged"]
 
 
 def _pr_id_from_webhook(payload: Dict[str, Any]):
     return payload["raw_payload"]["payload"]["number"]
+
+
+def _is_version_pr(payload: Dict[str, Any]):
+    return "version" in payload["raw_payload"]["payload"]["pull_request"]["title"]
+
+
+def _get_version(payload: Dict[str, Any]):
+    pr_title = payload["raw_payload"]["payload"]["pull_request"]["title"]
+    return re.search(r'version\s*([\d.]+)', pr_title).group(1)
```

Comment:
from another bot reference it seems , we can get the version from the branch the pr was opened from.
as kebechet creates a branch first and then creates a pr.
```suggestion
    release_version = payload["raw_payload"]["payload"]["pull_request"]["head"]["ref"]
    return release_version
```

Reply:
Yeah, I think this is a better approach. Titles and stuff are more likely to change. Branch names have been relatively static and contain just the release version.

Reply:
Ack! done